### PR TITLE
chore: removing vision unsupported samples

### PR DESCRIPTION
**Pull request by munkhuushmgl**
go/cloud-ai-2021-snippet-cleanup

Devsite:
cl/363755327 [submitted]

cl/363758676

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
Info on non-vision failures:
- Storage failure: #1272 
- Spanner, transient failure. We are retrying but in this case it wasn't enough. This should fall within normal expeced behaviour.
- Asset failure: #1278

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
There's one tag reported by snippet-bot as being deleted but used in docs.
And also Vision tests seem to be failing, most of them.

**Comment by munkhuushmgl** [comment]
> There's one tag reported by snippet-bot as being deleted but used in docs.
> And also Vision tests seem to be failing, most of them.

one region tag should be fixed by now.

cl/363982019

**Comment by amanda-tarafa** [comment]
Region tags look good.
Vision tests consistently failing though. See [here](https://source.cloud.google.com/results/invocations/aa056682-b048-4bb8-8695-3ade0d4ca275/targets/github%2Fdotnet-docs-samples%2Fvision%2Fapi%2FVisionTest%2FTestResults/tests) for instance.

**Comment by munkhuushmgl** [comment]
> Region tags look good.
> Vision tests consistently failing though. See [here](https://source.cloud.google.com/results/invocations/aa056682-b048-4bb8-8695-3ade0d4ca275/targets/github%2Fdotnet-docs-samples%2Fvision%2Fapi%2FVisionTest%2FTestResults/tests) for instance.

Looks like I did not delete tests related to removed samples.

**Comment by amanda-tarafa** [comment]
Still lots of [failures](https://source.cloud.google.com/results/invocations/43ce07a1-f06c-4695-b065-acc02453e7ab/targets/github%2Fdotnet-docs-samples%2Fvision%2Fapi%2FVisionTest%2FTestResults/tests) for Vision tests. Some files seem to be missing.

**Comment by munkhuushmgl** [comment]
@amanda-tarafa Should be good now.

File: vision/api/VisionTest/VisionTest.cs [hunk 1]
```diff
@@ -1,344 +0,0 @@
...
-// the License.
-
-using Google.Cloud.Storage.V1;
-using Google.Cloud.Vision.V1;
-using System;
-using System.IO;
-using System.Linq;
-using Xunit;
-
-namespace GoogleCloudSamples
```
Comment (amanda-tarafa) [inline vision/api/VisionTest/VisionTest.cs:22]: We need to keep the tests for the two samples remaining.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]

**Comment by amanda-tarafa** [comment]
F# failure is related to this PR, see #1289. I'll fix it.

**Comment by munkhuushmgl** [comment]
> F# failure is related to this PR, see #1289. I'll fix it.

how do I see test failures related to vision?

**Comment by amanda-tarafa** [comment]
I already fixed #1289 so no worries about it.

Vision runs in Kokoro CI Windows 3 and Kokoro CI Linux 3. You can click on `Details` link of those machines (on the widget at the end of the PR that shows checks passing and failing). And there you can click on the Vision related link.

**Comment by munkhuushmgl** [comment]
@amanda-tarafa PTAL something failed outside of my changes

**Comment by amanda-tarafa** [comment]
Failures are unrelated and being tracked:

* SQL Server: #1311

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Pull request merged**